src/main.ts

```diff
@@ -4,12 +4,14 @@ import * as inputHelper from "./input-helper"
 import * as thc from "typed-rest-client/HttpClient"
 
 import {ReleaseDownloader} from "./release-downloader"
+import {extract} from "./unarchive"
 
 async function run(): Promise<void> {
   try {
     const downloadSettings = inputHelper.getInputs()
     const authToken = core.getInput("token")
     const githubApiUrl = core.getInput("github-api-url")
+    const extract_assats = Boolean(core.getInput("extract"))
```

Comment:
The `core.getInput()` function reads input as a string, and the `Boolean()` function checks only if the input string is non-empty (when a string is provided as a param). As a result, `extract_assats` always remains true.

You may notice the following run tried to extract downloaded files even though the default value for `extract` was set to false in `action.yml`

<img width="858" alt="Screenshot 2023-04-25 at 11 27 47 am" src="https://user-images.githubusercontent.com/8011082/234252386-08e26713-60fd-4285-9652-7bf719db5173.png">

https://github.com/robinraju/release-downloader/pull/613/checks

We need to parse the boolean value here explicitly. I suggest a better way would be by adding an entry to `IReleaseDownloadSettings` (`download-settings.ts`) and parse it inside `input-helper.ts`. 

WDYT?

Reply:
On your side here! Is changed